### DISCUSSION TITLE
Loading german ("de") issue

### DISCUSSION THREAD

**Discussion by aidyai** [explosion/spaCy#7299] [Help: Coding & Implementations] [answered]
---------------------------------------------------------------------------
OSError                                   Traceback (most recent call last)
<ipython-input-15-40b84c767ef8> in <module>
      1 from spacy.lang.de.examples import sentences
----> 2 nlp = spacy.load("de_core_news_sm")
      3 
      4 
      5 docs = nlp.pipe(sentences)

/opt/conda/lib/python3.7/site-packages/spacy/__init__.py in load(name, disable, exclude, config)
     45     RETURNS (Language): The loaded nlp object.
     46     """
---> 47     return util.load_model(name, disable=disable, exclude=exclude, config=config)
     48 
     49 

/opt/conda/lib/python3.7/site-packages/spacy/util.py in load_model(name, vocab, disable, exclude, config)
    327     if name in OLD_MODEL_SHORTCUTS:
    328         raise IOError(Errors.E941.format(name=name, full=OLD_MODEL_SHORTCUTS[name]))
--> 329     raise IOError(Errors.E050.format(name=name))
    330 
    331 

OSError: [E050] Can't find model 'de_core_news_sm'. It doesn't seem to be a Python package or a valid path to a data directory.

**Comment by aidyai** [comment]
![error](https://user-images.githubusercontent.com/45211523/110105940-e0178380-7d5d-11eb-9dd7-3a9bef1aa95e.JPG)

**Comment by polm** [comment, marked as answer]
Hey, sorry you're having trouble getting started. Before you can use a model you have to download it, like this:

    python -m spacy download de_core_news_sm

Also, when posting an issue, please follow the template and include your spaCy version and other information.

**Reply by aidyai** [reply]
using German didn't work with the latest version, so I did go back to spacy version 2.3 and german did work

**Reply by aidyai** [reply]
You can try it out yourself  in case

**Reply by polm** [reply]
Works fine for me. Can you paste the output of `pip freeze` so we can check the versions of everything you have installed?

**Comment by svlandeg** [comment]
(Maintenance note: this question is well suited for the discussions forum, so I'm moving it there. While this issue gets closed, the discussion thread will remain open to continue the conversation if need be!)

**Reply by aidyai** [reply]
Thanks for this..........

I installed a previous version and it worked but version 3. has issues loading ("de")

**Comment by svlandeg** [comment]
Hi @AidyStark, if you're trying to run

```
nlp = spacy.load("de")
```
that actually won't work in spaCy v3. As there are multiple different German models, [spaCy v3](https://spacy.io/usage/v3#incompat-api) requires you to always specify the full name of the model, and not just a shortcut like "de". So you'll have to spell it out like 
```
nlp = spacy.load("de_core_news_sm")
```
